code/lib/cli/src/initiate.ts

```diff
@@ -369,6 +372,15 @@ export async function doInitiate(
     return { shouldRunDev: false };
   }
 
+  const foundGitIgnoreFile = await findUp('.gitignore', { cwd: process.cwd() });
```

Comment:
```suggestion
  const foundGitIgnoreFile = await findUp('.gitignore');
```

`cwd: process.cwd()` is the default and is therefore not necessary to define: https://www.npmjs.com/package/find-up#cwd